Description: Do not display donations information in the menu navigation
 of the documentation. Commenting out the relevant <li> tag prevents lintian
 from raising several privacy-breach-donation errors.
Author: Hugh McMaster
Bug-Debian: https://bugs.debian.org/873432
Forwarded: not-needed
Last-Update: 2021-10-11

--- a/ft2docs/docs/documentation.html
+++ b/ft2docs/docs/documentation.html
@@ -107,7 +107,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -122,7 +122,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/ft2faq.html
+++ b/ft2docs/docs/ft2faq.html
@@ -616,7 +616,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -631,7 +631,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/index.html
+++ b/ft2docs/docs/index.html
@@ -263,7 +263,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -278,7 +278,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/design/design-1.html
+++ b/ft2docs/docs/design/design-1.html
@@ -159,7 +159,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -174,7 +174,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/design/design-2.html
+++ b/ft2docs/docs/design/design-2.html
@@ -214,7 +214,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -229,7 +229,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/design/design-3.html
+++ b/ft2docs/docs/design/design-3.html
@@ -366,7 +366,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -381,7 +381,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/design/design-4.html
+++ b/ft2docs/docs/design/design-4.html
@@ -328,7 +328,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -343,7 +343,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/design/design-5.html
+++ b/ft2docs/docs/design/design-5.html
@@ -470,7 +470,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -485,7 +485,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/design/design-6.html
+++ b/ft2docs/docs/design/design-6.html
@@ -336,7 +336,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -351,7 +351,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/design/index.html
+++ b/ft2docs/docs/design/index.html
@@ -161,7 +161,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -176,7 +176,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/glyphs/glyphs-1.html
+++ b/ft2docs/docs/glyphs/glyphs-1.html
@@ -194,7 +194,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -209,7 +209,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/glyphs/glyphs-2.html
+++ b/ft2docs/docs/glyphs/glyphs-2.html
@@ -402,7 +402,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -417,7 +417,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/glyphs/glyphs-3.html
+++ b/ft2docs/docs/glyphs/glyphs-3.html
@@ -451,7 +451,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -466,7 +466,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/glyphs/glyphs-4.html
+++ b/ft2docs/docs/glyphs/glyphs-4.html
@@ -206,7 +206,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -221,7 +221,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/glyphs/glyphs-5.html
+++ b/ft2docs/docs/glyphs/glyphs-5.html
@@ -374,7 +374,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -389,7 +389,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/glyphs/glyphs-6.html
+++ b/ft2docs/docs/glyphs/glyphs-6.html
@@ -458,7 +458,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -473,7 +473,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/glyphs/glyphs-7.html
+++ b/ft2docs/docs/glyphs/glyphs-7.html
@@ -356,7 +356,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -371,7 +371,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/glyphs/index.html
+++ b/ft2docs/docs/glyphs/index.html
@@ -211,7 +211,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -226,7 +226,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/tutorial/index.html
+++ b/ft2docs/docs/tutorial/index.html
@@ -149,7 +149,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -164,7 +164,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/tutorial/step1.html
+++ b/ft2docs/docs/tutorial/step1.html
@@ -1025,7 +1025,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -1040,7 +1040,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/tutorial/step2.html
+++ b/ft2docs/docs/tutorial/step2.html
@@ -1541,7 +1541,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -1556,7 +1556,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
--- a/ft2docs/docs/tutorial/step3.html
+++ b/ft2docs/docs/tutorial/step3.html
@@ -120,7 +120,7 @@
 
 <div id="TOC">
   <ul>
-    <li class="funding">
+<!--    <li class="funding">
       <form action="https://www.paypal.com/cgi-bin/webscr"
             method="post"
             target="_top">
@@ -135,7 +135,7 @@
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
-    </li>
+    </li>-->
     <li class="primary">
       <a href="http://freetype.org/index.html">Home</a>
     </li>
